CHANGES.txt

```diff
@@ -23,6 +23,9 @@ Fixes
    crash in verbose mode. This prevents the programme from printing the message
    to stdout when using the command mode or executing ``--sysinfo``.
 
+ - Fixed an error caused by using the `\verbose` command before using `\connect`
```

Comment:
use double backticks for inline code
and please move on top of list

Reply:
Aha, thanks. Been a while since I did a changelog entry...